components/terraform/cognito.tf

```diff
@@ -0,0 +1,24 @@
+variable "user_pool_domain" {}
+variable "nationalarchives_callback_url" {}
+variable "nationalarchives_logout_url" {}
+
+module "etna-cognito" {
+    source = "./cognito"
+
+    cognito_user_pool_name = "nationalarchives-user-pool"
```

Comment:
Could we variables for the values. Makes copy and paste of code easier or changes in naming for the future.